src/components/ProductListItem/ProductListItem.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <li :id="id" class="items-container__item">
+   <div class="photo">
+    <img
+      class="item"
+      :src="url"
+      alt="Shop item"
```

Comment:
Consider having a unique alt for each of the photos.
Maybe this would work?
```suggestion
      :alt="`${title} photo`"
```